src/pages/AppProfile.vue

```diff
@@ -236,9 +241,17 @@ export default {
 
     function onUpdateOrganization() {
       try {
+        const size = Number(organisationDetails.value.size)
+        if (!Number.isFinite(size) || !Number.isSafeInteger(size) || size <= 0) {
+          organisationDetails.value.sizeErr = 'Invalid organization size.'
+          return
+        } else {
```

Comment:
Don't need the `else` here because of the early `return` inside `if`.

Reply:
No, the return is to shortcircuit when the size is invalid, but otherwise if you change it afterwards and the new size is **not** invalid, then we also need to clear the error, this is related to that.